.github/workflows/test-python.yml

```diff
@@ -0,0 +1,59 @@
+---
+name: test-python
+
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches:
+      - main
+      - renovate/**
+  pull_request:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 20
+    strategy:
+      matrix:
+        python-version:
+          - "3.8"
+          - "3.9"
+          - "3.10"
+          - "3.11"
+          - "pypy3.8"
```

Comment:
Is this similar to 3.8? Your matrix is gonna get rather large rather quickly

Reply:
It won't be - the suite of tests is pretty small.